jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -439,80 +444,122 @@ private void checkImageInTargetRegistry() {
                                 results
                                     .builtImagesAndContainerConfigurationPushResults
                                     .get()
-                                    .get(results.builtImages.get().get(0)))
+                                    .values()
```

Comment:
Do we really need this change? I was aware that we can always use `.iterator().next()` to retrieve a value from a single-entry map, but you've been so far consistently calling `Map.get()`. Mixing patterns makes it harder to understand code.